tasties_app/templates/tasties_app/base.html

```diff
@@ -33,27 +33,32 @@
     <nav class="navbar navbar-light bg-light">
 
       <!-- Logo -->
-      <a class="navbar-brand" id="logo" href="{% url 'index' %}">
+      <a class="navbar-brand" style="color: #10654e" href="{% url 'index' %}">
         <img
           src="{% static 'images/logo.png' %}"
+          width="30"
+          height="25"
           class="d-inline-block align-top"
           alt="tasties-logo"
+          style="margin-top: 5px"
```

Comment:
Didn't we agree in the previous PR you had, that all of those configurations are going to be in the CSS file?

Reply:
Yes, I made a mistake while resolving conflicts with this file.
Thanks for noticing.